test/ui-test/src/screens/StatusMainScreen.py

```diff
@@ -60,43 +56,19 @@ class StatusMainScreen:
     def __init__(self):
         verify_screen(MainScreenComponents.CONTACTS_COLUMN_MESSAGES_HEADLINE.value)
         
-    # Main screen is ready to interact with it (Splash screen animation not present and no banners on top of the screen)
+    # Main screen is ready to interact with it (Splash screen animation not present)
     def is_ready(self):
         self.wait_for_splash_animation_ends()
-        self.close_banners()
         verify(is_displayed(MainScreenComponents.CONTACTS_COLUMN_MESSAGES_HEADLINE.value), "Verifying if the Messages headline is displayed")
         
     def wait_for_splash_animation_ends(self, timeoutMSec: int = 10000):
         start = time.time()
         [loaded, obj] = is_loaded_visible_and_enabled(MainScreenComponents.SPLASH_SCREEN.value)
         while loaded and (start + timeoutMSec / 1000 > time.time()):
             log("Splash screen animation present!")
-            [loaded, obj] = is_loaded_visible_and_enabled(MainScreenComponents.SPLASH_SCREEN.value)            
+            [loaded, obj] = is_loaded_visible_and_enabled(MainScreenComponents.SPLASH_SCREEN.value, 1000)            
             sleep_test(0.5)
         verify_equal(loaded, False, "Checking splash screen animation has ended.")
-    
-    # It closes all existing banner and waits them to disappear:
-    def close_banners(self):
-        self.wait_for_banner_to_disappear(MainScreenComponents.CONNECTION_INFO_BANNER.value)
-        self.wait_for_banner_to_disappear(MainScreenComponents.UPDATE_APP_BANNER.value)
-        self.wait_for_banner_to_disappear(MainScreenComponents.SECURE_SEEDPHRASE_BANNER.value)
-
```

Comment:
I would not remove it since we could, in some point, want to test some banner functionality and / or also test that they really desappear, so.. Maybe we can still keep this code (with a comment?¿)

Reply:
I'm not sure how we could do that TBH. Since it's an env var that manages if we show the banner or not, I'm not sure if it's possible to change env vars in between tests.

Reply:
I guess we can do it as you said using maybe different test env vars and yeah, setting them in the hooks, for example.

Reply:
Anyway, I'm not  a fan of letting code sit as comments. Might as well remove if it's not used for now. We always have git history if we want to add back.

Often, the code changes and un-commenting doesn't even work because some apis or names have changed.